Mindee Invoice.ipynb

```diff
@@ -0,0 +1,417 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(parsed_data.http_response)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "minute-charger",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "26.04\n",
+      "4.95\n",
+      "30.99\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(parsed_data.invoice.total_excl.value)\n",
+    "print(parsed_data.invoice.total_tax.value)\n",
+    "print(parsed_data.invoice.total_incl.value)"
```

Comment:
Similarly, I'd suggest some formatting:
```python
print(f"Total excluding amount: {parsed_data.invoice.total_excl.value}€")
print(f"Total tax amount: {parsed_data.invoice.total_tax.value}€")
print(f"Total including amount: {parsed_data.invoice.total_incl.value}€")
```

Reply:
added your suggestion (minus the euro symbol - since some invoices may be in £ or $...)